values/gitea/gitea.gotmpl

```diff
@@ -110,18 +112,31 @@ gitea:
       DOMAIN: {{ $giteaDomain }}
       ROOT_URL: "https://{{ $giteaDomain }}/"
       DISABLE_SSH: true
-
+      HTTP_PORT: 3000
   metrics:
     enabled: true
     serviceMonitor:
       enabled: true
-      additionalLabels:
-        prometheus: system
+      relabelings:
+        - sourceLabels: [__name__]
+          targetLabel: prometheus
+          replacement: system
+  additionalConfigFromEnvs:
+    - name: GITEA__DATABASE__USER
```

Comment:
I couldn't find GITEA__DATABASE__USER anywher in the gitea chart, so I am just double checking that this env is supported by Gitea

Reply:
Any value for the `app.ini` configuration file can also be specified through environment variables. However it is only documented well on the Helm chart repo readme. That option existed already in earlier versions but was not applied to the init containers and therefore could not be used for the database credentials. I added a reference to the docs.